.github/workflows/ci.yml

```
@@ -17,6 +17,19 @@ jobs:
     - uses: actions/setup-node@v2
       with:
         node-version: ${{ matrix.node }}
+    - name: check dependencies
+      run: |
+        if cat yarn.lock | grep "@definitelytyped/"; then
```

isn't this equivalent?
```suggestion
        if grep "@definitelytyped/" yarn.lock; then
```

I don't understand the scope of this command -- isn't this the root yarn.lock? How does this check the package yarn.locks? Or is yarn.lock a convenient single point where all incorrect dependencies show up?

---

For workspaces, there's a single yarn.lock.

And yes, I'm bad at bash.